Cython/Compiler/PyrexTypes.py

```diff
@@ -5370,3 +5370,16 @@ def cap_length(s, max_prefix=63, max_len=1024):
         return s
     hash_prefix = hashlib.sha256(s.encode('ascii')).hexdigest()[:6]
     return '%s__%s__etc' % (hash_prefix, s[:max_len-17])
+
+def remove_cv_ref(tp, dont_remove_fakeref=True):
```

Comment:
Why not just `remove_fakeref=True`? Seems easier to read than `dont_remove_fakeref=False`, where the negations take a while to untangle.

In code that calls the function but does not care about "fakerefs", you can reduce the complexity by not mentioning them. The only time when we do care about "fakerefs" is when we want to keep them.